cmd/torusd/main.go

```diff
@@ -110,10 +113,34 @@ func configureServer(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "error parsing read-cache-size: %s\n", err)
 		os.Exit(1)
 	}
-	size, err = humanize.ParseBytes(sizeStr)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "error parsing size: %s\n", err)
-		os.Exit(1)
+	if strings.Contains(sizeStr, "%") {
+		var stat syscall.Statfs_t
```

Comment:
we probably should separate this into two different funcs: parsePercentage and getDiskSize.

for getDiskSize, do we want to be portable? current solution only works on unix i think.

not sure if it worth to introduce a dependency to handle it: https://github.com/StalkR/goircbot/tree/master/lib/disk

or we can use os.FileInfo and get os dependent field in Sys() https://golang.org/pkg/os/#FileInfo